dspace/config/submission-forms.dtd

```diff
@@ -55,3 +55,9 @@
  <!ATTLIST language value-pairs-name CDATA  #IMPLIED>  
 
  <!ELEMENT readonly (#PCDATA) >
+
+<!ELEMENT relationship-type (#PCDATA) >
+<!ELEMENT search-configuration (#PCDATA) >
+<!ELEMENT filter (#PCDATA) >
+<!ELEMENT relation-field (relationship-type, search-configuration, filter?, repeatable?, label, hint, linked-metadata-field?, required?)>
```

Comment:
Don't we need all the other elements of the `field` element here, too? At least `type-bind` and `visibility` are missing and maybe `language`, `style` and `readonly` (if this is supported for relation-fields).

Reply:
The `type-bind` and `visibility` can definitely be applied here.
Not sure what the others would possibly do for a relation-field

Reply:
`readonly` may only be useful, if we can define a predefined value with the item template, later. `language` may be used with alternative names. I am fine with the addition now, other elements can be added later if needed.